src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -203,17 +207,25 @@ public void sendNewDARCollectionMessage(Integer collectionId)
           sendList.put(dac.getName(), matchingDatasetsForDac);
         }
       }
-      sendNewDARRequestEmail(user, sendList, researcherName, collection.getDarCode());
+      // If the dar is not a progress report, use the DAR template else use the PR template.
+      if (dar.getProgressReport()) {
```

Comment:
We now have both `DarCollectionSummary.getProgressReport()` and `DataAccessRequest.getProgressReport()`. 

Could we add `getProgressReport()` to `DarCollection` as `getMostRecentDar().getProgressReport()`? Then we can use `DarCollection.getProgressReport()` in `DarCollectionSummary()` instead of having it compute it separately.

Reply:
I'm not sure I'm following the thought here.  I don't see where DarCollectionSummary currently uses DarCollection today.  Maybe we can walk through your idea.

In a similar vain though, I'm wondering if we can eliminate DarCollection.  As you'll see in my refactor it's clear that we're moving into spaces where DARs can be grouped together by an ID (collection ID or darCode).  I'm not sure we really need the DarCollection object anymore.

Reply:
> I'm wondering if we can eliminate DarCollection
I think we need to move in this direction ... will file a ticket.

Reply:
https://broadworkbench.atlassian.net/browse/DT-1665